praw/endpoints.py

```diff
@@ -105,6 +105,7 @@
     "mentions":                "message/mentions",
     "message":                 "message/messages/{id}/",
     "messages":                "message/messages/",
+    "modnotes":                "/api/mod/notes",
```

Comment:
```suggestion
    "mod_notes":               "/api/mod/notes",
```
Nit, what about naming it this?